packages/demo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "demo",
-  "version": "2.0.9",
+  "version": "2.0.10-alpha.1",
```

Comment:
```suggestion
  "version": "2.0.10",
```

Reply:
Instead of bumping this can you bump the version of `@web3-onboard/core` on line 36 below?

Reply:
Thank you! I've bumped it in https://github.com/blocknative/web3-onboard/pull/2057/commits/fc3c8316025f6b486bc13e83ea11713e28bb3f44